### ISSUE TITLE
ImportError on pandas import

### ISSUE THREAD

**Issue by IanSudbery** [CGATOxford/cgat#383]
Hi All,

When I try running any cgat script that requires pandas I get the following error:

```python-traceback
RuntimeError: module compiled against API version 0xb but this version of numpy is 0xa
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/mbp17idy/.conda/envs/sharc/lib/python2.7/site-packages/pkg_resources/__init__.py", line 570, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/home/mbp17idy/.conda/envs/sharc/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2687, in load_entry_point
    return ep.load()
  File "/home/mbp17idy/.conda/envs/sharc/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2341, in load
    return self.resolve()
  File "/home/mbp17idy/.conda/envs/sharc/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2347, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/home/mbp17idy/devel/CGATReport-0.3/CGATReport/build.py", line 40, in <module>
    from CGATReport import report_directive, clean, Utils
  File "/home/mbp17idy/devel/CGATReport-0.3/CGATReport/report_directive.py", line 21, in <module>
    from CGATReport import Config, Dispatcher, Utils, Cache, Component
  File "/home/mbp17idy/devel/CGATReport-0.3/CGATReport/Config.py", line 24, in <module>
    exec(compile(open("conf.py").read(), "conf.py", 'exec'))
  File "conf.py", line 18, in <module>
    import CGATPipelines.Pipeline as P
  File "/home/mbp17idy/devel/CGATPipelines/CGATPipelines/Pipeline/__init__.py", line 220, in <module>
    from CGATPipelines.Pipeline.Database import *
  File "/home/mbp17idy/devel/CGATPipelines/CGATPipelines/Pipeline/Database.py", line 33, in <module>
    from CGAT import Database as Database
  File "/home/mbp17idy/devel/cgat/CGAT/Database.py", line 14, in <module>
    from pandas import DataFrame
  File "/home/mbp17idy/.conda/envs/sharc/lib/python2.7/site-packages/pandas-0.20.3-py2.7-linux-x86_64.egg/pandas/__init__.py", line 35, in <module>
    "the C extensions first.".format(module))
ImportError: C extension: umpy.core.multiarray failed to import not built. If you want to import pandas from the source directory, you may need to run 'python setup.py build_ext --inplace --force' to build the C extensions first.
```

If I import pandas from an interactive python interpretor, it works fine. If I do the following, I get the error:

```
>>> __requires__ = 'CGATReport'
>>> from pkg_resources import load_entry_point
>>> load_entry_point('CGATReport', 'console_scripts', 'cgatreport-build')()
```

(I know this is a CGATReport Error, but it is also coming through, for eg bam2geneprofile)

Any ideas appreciated!

**Comment by sebastian-luna-valero** [comment]
Hi Ian,

After sourcing your environment, could you please send the output of:
```
which python

python -c "import pandas"
```

Also, it might be a problem with the `LD_LIBRARY_PATH`, so please have a look at:
```
env | grep LD_LIBRARY_PATH
```

Best regards,
Sebastian

**Comment by IanSudbery** [comment]
Hi Sebastian,
I think we've sorted it out.

Turns out that we had two versions of both pandas and numpy installed in the same `site-packages` directory. One came from the conda build. The other, I think, came from trying to install CGATReport from the github to get an up-to-date fix for something. Doing this caused it to redownload and install many of its dependencies from source, including different versions of numpy and pandas, leaving two installed versions. The naitive python import function was picking up one version, but pkg_resources was picking up the other. 

We solved this by doing

```
conda uninstall --force pandas
conda uninstall --force numpy
pip uninstall pandas
pip uninstall numpy
rm <path_to_site_libs>/pandas* -r
rm <path_to_site-libs>/numpy* -r

conda install pandas
```
